content/docs/deployment/csminstallationwizard/src/static/js/tests/utility.test.js

```diff
@@ -49,6 +49,7 @@ const CONSTANT_PARAM = {
 	CSM_HELM_V1102: "1.3.2",
```

Comment:
We should remove this if we are removing the support for `CSM_HELM_V1102`.

Reply:
Removed it. Thanks!